src/cityreader/cityreader.py

```diff
@@ -1,6 +1,13 @@
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
 
+    def __repr__(self):
+        return f"<'City': {self.name}, {self.lat}, {self.lon}>"
```

Comment:
This is a somewhat unwritten rule of `__repr__` in Python, but generally you want your `__repr__` to return a string that could be passed to `eval` and create an identical class.

For example, if in Python you would create a City object like so:

```python
>>> City(name="Seattle", lat=47.6217, lon=-122.3238)
```

Then you would want a `__repr__` that looks like this:

```python
def __repr__(self):
    return f"City(name='{self.name}', lat={self.lat}, lon={self.lon})"
```

It is in no way important, but I feel like sometimes the difference between `__repr__` and `__str__` got confusing for me early on.